core/app/models/spree/product/scopes.rb

```diff
@@ -23,7 +23,7 @@ def self.add_simple_scopes(scopes)
         # We should not define price scopes here, as they require something slightly different
         next if name.to_s.include?("master_price")
         parts = name.to_s.match(/(.*)_by_(.*)/)
-        self.scope(name.to_s, -> { order("#{Product.quoted_table_name}.#{parts[2]} #{parts[1] == 'ascend' ?  "ASC" : "DESC"}") })
+        self.scope(name.to_s, -> { order(Spree::Product.arel_table[parts[2].to_sym].send(parts[1] == 'ascend' ?  :asc : :desc)) })
```

Comment:
Line is too long. [130/120]<br>Unnecessary spacing detected.<br>Redundant `self` detected.<br>Operator `?` should be surrounded by a single space.